toolchain/check/pending_block.h

```diff
@@ -39,52 +39,52 @@ class PendingBlock {
     size_t size_;
   };
 
-  auto AddNode(SemIR::Node node) -> SemIR::NodeId {
-    auto node_id = context_.nodes().AddInNoBlock(node);
-    nodes_.push_back(node_id);
-    return node_id;
+  auto AddInst(SemIR::Inst inst) -> SemIR::InstId {
+    SemIR::InstId inst_id = context_.insts().AddInNoBlock(inst);
```

Comment:
Preserve `auto` for now:
```suggestion
    auto inst_id = context_.insts().AddInNoBlock(inst);
```